tests/Concise/Mock/BuilderPropertyTest.php

```diff
@@ -18,6 +18,44 @@ public function __set($name, $value)
     }
 }
 
+class ParentClass
+{
+    private /** @noinspection PhpUnusedPrivateFieldInspection */
```

Comment:
`PhpUnusedPrivateFieldInspection` can be removed now since all the private attributes are accessed from methods.